packages/gas-fee-controller/src/GasFeeController.ts

```diff
@@ -548,7 +548,7 @@ export class GasFeeController extends PollingController<
     }, this.intervalDelay);
   }
 
-  async executePoll(networkClientId: string): Promise<void> {
+  async _executePoll(networkClientId: string): Promise<void> {
```

Comment:
What do you think about copying the same JSDoc as from TokenListController for this method for consistency with the other controllers?

Reply:
done here: https://github.com/MetaMask/core/pull/1810/commits/8536555a04fe512dfdffdf93875b4c56cd22dfd4